src/client/app/helpers/trusted.snippet.helper.ts

```diff
@@ -1,3 +1,5 @@
+import { storage } from '../helpers';
+
 class TrustedSnippet {
     private _trustedSnippetsKey = 'trusted_snippets';
```

Comment:
would prefix this with `playground` check our other storage keys to see what's the prefix.